apollo-router/src/plugins/traffic_shaping/deduplication.rs

```diff
@@ -124,35 +124,39 @@ where
                     let authorization_cache_key = request.authorization.clone();
                     let id = request.id.clone();
                     let cache_key = ((&request.subgraph_request).into(), authorization_cache_key);
-                    let res = {
+                    let (res, hdl) = {
```

Comment:
What does hdl mean?

Reply:
handle

Reply:
You can take the person out of unix, but you can't take the unix out of.... :)

Reply:
I did `s/hdl/handle/g` in https://github.com/apollographql/router/pull/7347/commits/e5ecdc863869e801cabca5432c74e810eb6dac71